### PR TITLE
[Notifier] Streamline composer.json files

### PR DESCRIPTION
**Pull request by OskarStark**
| Q             | A
| ------------- | ---
| Branch?       | 5.1
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | ---
| License       | MIT
| Doc PR        | ---

### CODE REVIEW

File: src/Symfony/Component/Notifier/Bridge/Firebase/composer.json [hunk 1]
```diff
@@ -2,7 +2,10 @@
     "name": "symfony/firebase-notifier",
     "type": "symfony-bridge",
     "description": "Symfony Firebase Notifier Bridge",
-    "keywords": ["firebase", "notifier"],
+    "keywords": [
+        "firebase",
+        "notifier"
+    ],
```
Comment (fabpot) [inline src/Symfony/Component/Notifier/Bridge/Firebase/composer.json:8]: I think we have all keywords on the same line everywhere else, I would keep it that way.

**Review by fabpot — COMMENTED** [review]
I don't think these changes are needed to be honest

File: src/Symfony/Component/Notifier/Bridge/Firebase/composer.json [hunk 2]
```diff
@@ -21,7 +24,9 @@
         "symfony/notifier": "^5.0,<5.2"
     },
     "autoload": {
-        "psr-4": { "Symfony\\Component\\Notifier\\Bridge\\Firebase\\": "" },
+        "psr-4": {
+            "Symfony\\Component\\Notifier\\Bridge\\Firebase\\": ""
+        },
```
Comment (OskarStark) [inline src/Symfony/Component/Notifier/Bridge/Firebase/composer.json:29]: @fabpot I agree with the keywords, but shall we keep this in one line or readable like this. In the 5.2 branch we have it mixed 🤔 

If you don't care, please close the PR 👍 Thanks

**Comment by derrabus** [comment]
If we really want to streamline the CS of our composer.json files, we should rather use a tool like `ergebnis/composer-normalize` and run it as GitHub Action. But I don't feel a strong need to do that either, I must admit. 😉

**Comment by nicolas-grekas** [comment]
Thank you but this is not consistent with the composer.json files in the other components.
Also, we have better to do ;)

**Pull request closed without merging**